hack/tools/requirements.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o nounset
+
+HELM_VERSION=v3.10.3
+KUBE_VERSION=1.22.0
```

Comment:
if we don't need kubuilder we don't need it

Reply:
it contains the kubectl installation.

Reply:
just for kubectl install kubebuilder, ....
https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/

Reply:
@roytman I changed that Thanks